docs/loading_environment_variables.md

```diff
@@ -0,0 +1,160 @@
+# Loading environment variables
+
+imgproxy can load environment variables from various sources such as:
+
+* [Local file](#local-file)
```

Comment:
```suggestion
* [Local file](#local-file)
```
This should either be "a local file" or "local files", whichever is more correct. I think usually the first one.